src/pages/Facility/services/FacilityServices.tsx

```
@@ -12,6 +12,64 @@
 import { ServiceCard } from "@/pages/Facility/settings/healthcareService/ServiceCard";
 import healthcareServiceApi from "@/types/healthcareService/healthcareServiceApi";
 
+type DuoToneIconName = keyof typeof duoToneIcons;
+
+function ServiceCard({
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ServiceCard'. Consider to delete it or rename it.

'ServiceCard' is defined here:


(lint/suspicious/noRedeclare)